peoplehr/client.bal

```diff
@@ -0,0 +1,260 @@
+    @display{label: "Get All Employee Detail"}
+    isolated remote function getAllEmployeeDetail(@display{label: "Request Detail"}AllEmployeeDataRequest payload) 
+                                                 returns GetAllEmployeeDetailResponse|error {
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        json authentication =  {APIKey: self.config.apiKey, Action: "GetAllEmployeeDetail"}; 
+        request.setJsonPayload(check jsonBody.mergeJson(authentication));
+        return self.basicClient->post(EMPLOYEE, request, targetType = GetAllEmployeeDetailResponse);
+    }
+
+    # Gets employee detail by id
+    #
+    # + payload - Request details on an employee
+    # + return - Successful response or an error
+    @display{label: "Get Employee Detail By Id"}
+    isolated remote function getEmployeeDetailById(@display{label: "Employee Request Detail"} EmployeeDetailRequest 
+                                                    payload) returns GetEmployeeDetailByIdResponse|error {
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        json authentication =  {APIKey: self.config.apiKey, Action: "GetEmployeeDetailById"};
+        request.setJsonPayload(check jsonBody.mergeJson(authentication)); 
+        return self.basicClient->post(EMPLOYEE, request, targetType = GetEmployeeDetailByIdResponse);
+    }
+
+    # Creates new employee
+    #
+    # + payload - Request details on an new employee
+    # + return - Successful response or an error
+    @display{label: "Create New Employee"}
+    isolated remote function createNewEmployee(@display{label: "Employee Detail"} NewEmployeeRequest|json payload)
+                                               returns CreateNewEmployeeResponse|error {
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        json authentication =  {APIKey: self.config.apiKey, Action: "CreateNewEmployee"};
+        request.setJsonPayload(check jsonBody.mergeJson(authentication)); 
+        return self.basicClient->post(EMPLOYEE, request, targetType = CreateNewEmployeeResponse);
+    }
+
+    # Updates employee details.
+    #
+    # + payload - Request details on an employee
+    # + return - Successful response or an error
+    @display{label: "Update Employee Detail"}   
+    isolated remote function updateEmployeeDetail(@display{label: "Employee Detail"} EmployeeUpdateRequest|json payload) 
+                                                  returns UpdateEmployeeDetailResponse|error {
+        http:Request request = new;
+        json jsonBody = check payload.cloneWithType(json);
+        json authentication =  {APIKey: self.config.apiKey, Action: "UpdateEmployeeDetail"}; 
+        request.setJsonPayload(check jsonBody.mergeJson(authentication));
+        return self.basicClient->post(EMPLOYEE, request, targetType = UpdateEmployeeDetailResponse);
+    }
+
```

Comment:
Shall we reorganize the remote methods to be in CRUD order? That's the order the drop down at Choreo would list down the operations. If you see update operation in the dropdown first, it is kind of weird.